packages/x-charts/src/hooks/useTicks.ts

```diff
@@ -145,7 +145,7 @@ export function useTicks(
       }));
     }
 
-    if (scale.domain().length === 0 || scale.domain()[0] === scale.domain()[1]) {
+    if (scale.domain().length === 0 || !scale.domain().every(Number.isFinite)) {
```

Comment:
Here you removed the second condition. I keep the check on infinite to avoid plotting the grid if the axis has an inifinit domain. For example the y-axis of a line chart without data

Reply:
Looks like it's time that causes issue. Because the `Number.isFinite()` returns false for dates. So the code was doing a confusion between `Infinity` and `Date(...)`

Reply:
Gotta push that new `isInfinity` function though 🤣